src/test/java/uk/gov/companieshouse/overseasentitiesapi/utils/DataSanitisationTest.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class DataSanitisationTest {
+
+    @Test
+    void testMakeStringSafeForLogging() {
+        DataSanitisation dataSanitisation = new DataSanitisation();
+        String sanitisedInput = dataSanitisation.makeStringSafeForLogging("abc\t\nabc");
```

Comment:
make the string to be sanitised > 50 chars as well to test that it truncates it

Reply:
Added another test for this.